internal/vpn/os.go

```diff
@@ -0,0 +1,204 @@
+package vpn
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+	"net"
+	"os"
+	"os/exec"
+	"strconv"
+)
+
+// SetupTUN sets the allocated TUN interface up, setting its IP, gateway, netmask and MTU.
+func SetupTUN(ifcName, ip, netmask, gateway string, mtu int) error {
+	return run("/sbin/ifconfig", ifcName, ip, gateway, "mtu", strconv.Itoa(mtu), "netmask", netmask, "up")
+}
+
+// NetworkIfcGateway gets gateway of the network interface with name `ifcName`.
+func NetworkIfcGateway(ifcName string) (net.IP, error) {
+	cmd := fmt.Sprintf(gatewayForIfcCMDFmt, ifcName)
+	outBytes, err := exec.Command("/bin/bash", "-c", cmd).Output() //nolint:gosec
```

Comment:
Would `/bin/sh` instead of `/bin/bash` work? Sometimes, `bash` is not pre-installed.